grunt_tasks/build.js

```diff
@@ -103,12 +90,24 @@ module.exports = function (grunt) {
                         'clients/web/src/stylesheets/apidocs/*.styl'
                     ]
                 }
+            }
+        },
+
+        concat: {
+            options: {
+                stripBanners: {
+                    block: true,
+                    line: true
+                }
             },
-            ext: {
+            ext_css: {
                 files: {
                     'clients/web/static/built/ext.min.css': [
-                        'node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.css',
-                        'node_modules/jsoneditor/dist/jsoneditor.css'
+                        'node_modules/bootstrap/dist/css/bootstrap.min.css',
+                        'node_modules/bootstrap-switch/dist/css/bootstrap3/bootstrap-switch.min.css',
+                        'node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
+                        'node_modules/jsoneditor/dist/jsoneditor.min.css',
+                        'node_modules/as-jqplot/dist/jquery.jqplot.min.css'
```

Comment:
We could change it to just concatenate the minified 3rd party JS too, but I doubt it will make a difference (except perhaps to make builds marginally faster), so I left it with the status quo.

The only other argument I can think of for leaving it the same is that perhaps some of the 3rd party libraries used a minifier that produced inferior output, and thus Uglify can do a better job, but this is a bit of a stretch.

I'll change it if you can think of a good reason (even just consistency).